### PR TITLE
Don't generate a method if a parent already has it

### PR DESCRIPTION
**Pull request by EPashkin**
Part of #139 

WIP

### CODE REVIEW

**Comment by EPashkin** [comment]
@GuillaumeGomez, seem this PR done.
I added PR in gtk with results and in example for build fix after change.

File: src/analysis/record.rs [hunk 1]
```diff
@@ -84,7 +85,7 @@ pub fn new(env: &Env, obj: &GObject) -> Option<Info> {
         deprecated_version: deprecated_version,
         cfg_condition: obj.cfg_condition.clone(),
     };
-    
+
```
Comment (GuillaumeGomez) [inline src/analysis/record.rs]: Oh nice one! ;)

File: src/codegen/general.rs [hunk 2]
```diff
@@ -118,6 +118,16 @@ pub fn version_condition_string(env: &Env, version: Option<Version>, commented:
     }
 }
 
+pub fn not_version_condition(w: &mut Write, version: Option<Version>, commented: bool,
+                         indent: usize) -> Result<()> {
```
Comment (GuillaumeGomez) [inline src/codegen/general.rs]: The indent is broken here.

**Comment by GuillaumeGomez** [comment]
Seems good to me. As always, when you feel ready, just merge it. Thanks for your work!

**Pull request merged**